main/settings.py

```diff
@@ -31,6 +31,8 @@
     "crispy_forms",
     # M05 App "Hackathon" added
```

Comment:
Somehow missed this - can these comments be removed on the various lines? Not needed ultimately.